mesh/mesh_initializer.py

```diff
@@ -0,0 +1,40 @@
+__doc__ = """ Mesh Initializer using Pyvista """
+
+import pyvista as pv
+import numpy as np
+
+
+class MeshInitialize():
+    """
+    This Mesh Initializer class uses pyvista to import mesh files in the
+    STL or OBJ file formats and initializes the necessary mesh information.
+
+    How to initialize a mesh?
+    -------------------------
+
+    mesh = MeshInitialize(r('<filepath>'))
+
+    PS: Please be sure to add .stl / .obj at the end of the filepath, if already present, ignore.
+    """
+    def __init__(self, filepath):
+        self.mesh = pv.read(filepath)
+        self.pyvista_faces = self.mesh.faces
```

Comment:
why do you name as pyvista_faces and not faces?

Reply:
I named it pyvista_faces/points to specify that it's the faces array from pyvista library, I later used this array and established a self.faces array that stores the data in the desired format and dimension as requested by Ali in #270.

Reply:
I have added a detailed explanation of the function and class works as comments in the next commit; I will push that soon after this PR is resolved/merged.